src/repl/session.rs

```diff
@@ -679,6 +680,47 @@ impl Session {
         output.append(&mut result);
     }
 
+    #[cfg(feature = "cli")]
+    fn reload(&mut self, output: &mut Vec<String>) {
+        self.asts.clear();
+        self.contracts.clear();
+        self.costs_reports.clear();
+        self.coverage_reports.clear();
+        self.executed.clear();
+        self.initial_contracts_analysis.clear();
+        self.interpreter =
+            ClarityInterpreter::new(
+                self.interpreter.get_tx_sender(),
+                self.settings.repl_settings.clone());
+        let contracts = self.settings.initial_contracts.clone();
+        self.settings.initial_contracts.clear();
+
+        for existing in contracts {
+            let reloaded = match fs::read_to_string(&existing.path) {
+                Ok(xs) => xs,
```

Comment:
How about a more clear variable name here, like `code`.

Reply:
Oh my Haskell 👀 — I apologize for this. Yes, `code` looks better.